script.js

```diff
@@ -0,0 +1,74 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const getData = async () => {
+    const response = await fetch("https://jsonplaceholder.typicode.com/users");
+    const users = await response.json();
+    return users;
+  };
+
+  const createTable = async (thNames = ["id", "name", "phone", "website"]) => {
+    const table = document.createElement("table");
+    table.style.cssText = `
+        width: 1200px;
+        background: gray;
+        padding: 20px;
+        margin: 0 auto;
+        text-align: center;
+        border: 2px solid black;
+    `;
+    const caption = document.createElement("caption");
+    const thead = document.createElement("thead");
+    const trHead = document.createElement("tr");
+    const tbody = document.createElement("tbody");
```

Comment:
Make a helper function that gets tag name, styles, and text content as parameters and returns an element.